tests/kickstart_tests/user.ks

```diff
@@ -0,0 +1,88 @@
+# Set basic URL
+url --url="http://dl.fedoraproject.org/pub/fedora/linux/development/$releasever/$basearch/os/"
+
+# perform install
+install
+
+# Set network - automatical DHCP
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+
+# Remove whole disk info and recreate it.
+clearpart --all
+autopart --type=lvm
+
+# Set locales and Keyboard.
+keyboard us
+lang en
+timezone America/New_York
+
+# set users. Root first, others next
+rootpw qweqwe
+
+## TEST CREATE USER
+# First, we need to create a group where we can assign user..
+group --name=kosygroup --gid=5001
+
+# ..then create the user and assign him/her to group we created before.
+user --name=kosieh --gecos="Kosieh Barter" --homedir=/home/kbarter --password="$6$QsJCB9E6geIjWNvn$UZLEtnHYgKmFgrPo0fY1qNBc/aRi9b01f19w9mpdFm9.MPblckUuFYvpRLSzeYeR/6lO/2uY4WtjhbryC0k2L/" --iscrypted --shell=/bin/bash --uid=4001 --gid=5001
+
+# After all, shut down the machine.
+shutdown
+
+%packages
+%end
+
+%post
+
+## TEST CREATE USER CHECK
+
+# First, check if the group is properly set up.
+cat /etc/group | grep 5001
+if [[ $? -ne 0 ]]; then
+    echo "FAILED: Group failed to create." >> /root/RESULT
+fi
+
+# If it succeeds, lets go to higher level. Check, if the user is present..
```

Comment:
Remove 'If it succeeds, lets go yo higher level.' and similar below. When you write comments use rule: shorter is better.
This information isn't mandatory for the reader.